src/Invoice.php

```diff
@@ -2,7 +2,7 @@
 
 namespace LaravelDaily\Invoices;
 
-use Barryvdh\DomPDF\Facade as PDF;
+use Barryvdh\DomPDF\Facade\Pdf;
```

Comment:
needed to be updated because of [upstream changes](https://github.com/barryvdh/laravel-dompdf/pull/899)